server/remote/gcal/get_default_calendar_view_test.go

```diff
@@ -0,0 +1,138 @@
+package gcal
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/api/calendar/v3"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/remote"
+)
+
+func createMinimalCalendarEvent() calendar.Event {
+	return calendar.Event{
+		Summary: "test summary",
+		Start: &calendar.EventDateTime{
+			DateTime: "2023-08-01T00:01:02Z",
+			TimeZone: "UTC",
+		},
+		End: &calendar.EventDateTime{
+			DateTime: "2023-08-01T00:01:02Z",
+			TimeZone: "UTC",
+		},
```

Comment:
Just to be more realistic, can we have the end time be later?

Reply:
Did you want to implement this change?

Reply:
I recall doing it, that's why I resolved this... 🤔